R/util.R

```diff
@@ -43,6 +43,14 @@ util.apply_columns <- function(df, fun, ...){
     )
 }
 
+util.duplicated <- function(x) {
+  # returns a boolean of length x indicating whether each value in x is
+  # duplicated anywhere in x
```

Comment:
Nit: it's nice to include proper english grammar, punctuation, and capitalization wherever possible. They make prose more readable, they also make comments more readable.

Furthermore, the difference between this and `base::duplicated` is subtle enough to warrant comment. I suggest:

```
  # Returns a boolean indicating whether each value is
  # duplicated anywhere in x. This differs from base::duplicated, which
  # returns FALSE for the first of a set of duplicated values.
  #
  # Example: util.duplicated(c(1, 2, 1)) returns c(T, F, T).
```